app/bot.py

```diff
@@ -246,7 +253,10 @@ def advance_conversation(root_id: int, form_link: str) -> str:
 
```

Comment:
Curious about the use of 'elif' here with no apparent 'if' statement previously

Reply:
That is a monster function and I would defer to Rowen on that, as he is the one who originally wrote it. I just added my function where he told me. 🤣

Reply:
That's fair! I just wasn't sure about that one.  That was more of a clarity question than anything :joy: